pkgs/misc/emulators/stella/default.nix

```diff
@@ -7,19 +7,22 @@
 
 stdenv.mkDerivation rec {
   pname = "stella";
-  version = "6.5.3";
+  version = "6.6";
 
   src = fetchFromGitHub {
     owner = "stella-emu";
     repo = pname;
     rev = version;
-    hash = "sha256-Y9rEh9PZalQNj+d7OXN/8z5P8Hti4R3c2RL1BY+J1y4=";
+    hash = "sha256-+ZvSCnnoKGyToSFqUQOArolFdgUcBBFNjFw8aoVDkYI=";
   };
 
-  nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ SDL2 ];
+  nativeBuildInputs = [
+    pkg-config
+  ];
 
-  enableParallelBuilding = true;
```

Comment:
Why did you disable enableParallelBuilding?

Reply:
Playing safe and a bit paranoid.
Stella uses a custom builder that "emulates" autotools. A parallel building can be unpredictable.

Reply:
parallel building can speed up build times by a lot which also benefits hydra if the jobs are faster done. If the script is really incompatible it should be explicitly disabled with a comment so that it is kept disabled when enableParallelBuilding is switched to true by default.